src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.c

```diff
@@ -65,118 +136,1921 @@ const ActorInit En_Kaizoku_InitVars = {
 };
 
-#endif
+u32 D_80B8ABF8[] = {
+    0x0600F8E4, 0x0600EF9C, 0x0600E1C8, 0x0600DBE4, 0x060058B8, 0x06004860, 0x06002730,
+    0x06001578, 0x06001390, 0x060003CC, 0x06000F5C, 0x0600E8BC, 0x0600ED1C, 0x06005644,
+    0x0600F288, 0x060043E4, 0x06003A3C, 0x06002BA0, 0x06001E9C,
+};
+
+u8 D_80B8AC44[] = {
+    0x00, 0x00, 0x02, 0x00, 0x00, 0x02, 0x02, 0x02, 0x02, 0x02,
+    0x02, 0x02, 0x00, 0x02, 0x02, 0x02, 0x02, 0x00, 0x02, 0x00,
+};
+
+Color_RGBA8 D_80B8AC58 = { 0xC8, 0xA0, 0x78, 0xFF };
+Color_RGBA8 D_80B8AC5C = { 0x82, 0x5A, 0x32, 0xFF };
+
+Vec3f D_80B8AC60 = { 0.0f, -1.5f, 0.0f };
+Vec3f D_80B8AC6C = { 0.0f, -0.2f, 0.0f };
+Vec3f D_80B8AC78 = { 300.0f, 0.0f, 0.0f };
+Vec3f D_80B8AC84 = { 0.0f, -3000.0f, 0.0f };
+Vec3f D_80B8AC90 = { 400.0f, 0.0f, 0.0f };
+Vec3f D_80B8AC9C = { 1600.0f, -4000.0f, 0.0f };
+Vec3f D_80B8ACA8 = { -3000.0f, -2000.0f, 1300.0f };
+Vec3f D_80B8ACB4 = { -3000.0f, -2000.0f, -1300.0f };
+Vec3f D_80B8ACC0 = { 1000.0f, 1000.0f, 0.0f };
+
+u32 D_80B8ACCC[] = {
+    0x060093B8, 0x06009978, 0x0600A0F8, 0x06009978, 0x00000000,
+};
+
+extern AnimationHeader D_060058B8;
+
+extern FlexSkeletonHeader D_0600D828;
+
+void EnKaizoku_Init(Actor* thisx, PlayState* play) {
+    EnKaizoku* this = THIS;
+    s32 pad;
+    Player* player = GET_PLAYER(play);
+    EffectBlureInit1 blureInit;
+
+    this->actor.hintId = 0x54;
+    this->actor.targetMode = 3;
+    this->actor.colChkInfo.mass = 80;
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawFeet, 0.0f);
+    this->unk_2CA = this->actor.world.rot.z;
+    this->actor.colChkInfo.health = 8;
+    this->unk_2D4 = this->actor.params & 0x3F;
+    this->unk_2BC = (this->actor.params >> 6) & 0x7F;
+
+    if (this->unk_2CA >= 2) {
+        this->unk_2CA = 0;
+    } else if (this->unk_2CA == 2) {
+        this->unk_2CA = 0;
+    }
+
+    this->unk_2EC = this->actor.world.rot.z;
+    this->actor.world.rot.z = 0;
+    this->actor.colChkInfo.damageTable = &D_80B8AB3C;
+    SkelAnime_InitFlex(play, &this->skelAnime, &D_0600D828, &D_060058B8, this->jointTable, this->morphTable, 24);
+    Collider_InitAndSetCylinder(play, &this->unk_3D4, &this->actor, &D_80B8AB7C);
+    Collider_InitAndSetQuad(play, &this->unk_420, &this->actor, &D_80B8ABA8);
+    blureInit.p1StartColor[0] = blureInit.p1StartColor[1] = blureInit.p1StartColor[2] = blureInit.p1StartColor[3] =
+        blureInit.p2StartColor[0] = blureInit.p2StartColor[1] = blureInit.p2StartColor[2] = blureInit.p1EndColor[0] =
+            blureInit.p1EndColor[1] = blureInit.p1EndColor[2] = blureInit.p2EndColor[0] = blureInit.p2EndColor[1] =
+                blureInit.p2EndColor[2] = 255;
+    blureInit.p2StartColor[3] = 64;
+    blureInit.p1EndColor[3] = blureInit.p2EndColor[3] = 0;
+    blureInit.elemDuration = 8;
+    blureInit.unkFlag = 0;
+    blureInit.calcMode = 2;
+    Effect_Add(play, &this->unk_3D0, EFFECT_BLURE1, 0, 0, &blureInit);
+    Actor_SetScale(&this->actor, 0.0125f);
+    this->actor.flags |= 0x8000000;
```

Comment:
```suggestion
    this->actor.flags |= ACTOR_FLAG_8000000;
```
We have actor flags now, unnamed but at least you don't have to look them up.